### ISSUE TITLE
Appropriate way to close preview-tabbed?

### ISSUE THREAD

**Issue by hkdd** [jarun/nnn#721]
#### Environment details (Put `x` in the checkbox along with the information)

[Fedora 32] Operating System:
[x] Desktop Environment: dwm
[x] Terminal Emulator: urxvt
[x] Shell: bash
[x] Custom desktop opener (if applicable):xdg-open
[] Program options used:
[x] Configuration options set: da
[ ] Issue exists on `nnn` master

#### Exact steps to reproduce the issue
I cannot figure out the appropriate way to close the preview-tabbed window.  If it's an image, using "q" seems to just close the image preview but leaves the terminal open.  I can use dwm to kill the window, but then if I try to preview-tabbed again in the same sessions I just get a blank window.  I tried re-running the plugin to see if it worked as a toggle, but it just launches a new, third window, which is blank.  If I press "q" when previewing an image, the window title changes to "tabbed-0.6" if that's helpful at all.  

I've checked the man pages, googled around for it, the "?" screen and even tried to look through the preview-tabbed plugin script but cannot figure out the appropriate way to close it.

**Comment by jarun** [comment]
You'll have to press `^C`.

@leovilok can we document this in the plugin?

**Comment by hkdd** [comment]
Sorry, I should have said I tried ^C as well and that didn't do anything.  But if that should work I'll do some more poking around and see if I can tell why it is not for me.  ^C works fine in the terminal otherwise.  Also interesting is if I exit nnn completely it still leaves the preview-tabbed open.

**Comment by jarun** [comment]
Reopened the issue.

@leovilok can you please take a look?

**Comment by 0xACE** [comment]
^C within the preview tabbed pane. Otherwise it may remain open if someone is using the fifo file i guess

**Comment by hkdd** [comment]
It's actually launching a new terminal and ^C in either doesn't have an effect.  Should it?

**Comment by leovilok** [comment]
I'm quite busy, but I'll take a look.

**Comment by leovilok** [comment]
It *works for me*™ by closing the window in `dwm` (tested in `nnn` master). If I re-fire the plugin, the window is initially blank, but updates on first cursor move.

I agree that quitting the program inside tabbed and seing the bare tabbed window feels weird ... I'm not sure about how it should behave and how to implement that, and I don't have time to hack on this right now.

**Comment by leovilok** [comment]
> Also interesting is if I exit nnn completely it still leaves the preview-tabbed open.

I don't have this issue either, it should close on FIFO writer close.

**Comment by hkdd** [comment]
Yeah that behavior all checks out for me.  If I send dwm killclient to the preview window, then re-launch the plugin (;p for me) it is initially blank, but if I go up/down to another image or view, it will update.

I tested it again and now it is closing the preview window if I quit out of nnn.  Not sure how I got it to hang up previously. 

(Also totally unrelated but I'm blown away by how good nnn is.  I never really took the time previously to learn all the features and it is just amazingly powerful and efficient.)
